fa/cheatsheet-deep-learning.md

```diff
@@ -210,32 +211,31 @@
 
 **36. γ∈[0,1[ is the discount factor**
 
-&#10230;
+γ∈[0,1]  عامل تنزیل است
```

Comment:
factor translation is `عامل`.
But I think using something like `نرخ` or even `ضریب` is more understandable in RL context. 
other's opinion is needed here!

Reply:
The discount factor `γ` is a like metaparameter tuned by the user and represents how much future events lose their value according to how far away in time they are. since it's a constant quantity placed before and multiplying the variable i'd stick with  `ضریب`